src/components/button/JustifyCenterButton.tsx

```diff
@@ -0,0 +1,26 @@
+// Chakra imports
+import { useColorModeValue, Button, Tooltip } from "@chakra-ui/react";
+import { MdFormatAlignJustify } from "react-icons/md";
+// Custom components
+
+export default function Default() {
+	const textColor = useColorModeValue("secondaryGray.900", "white");
+	const boxBg = useColorModeValue("secondaryGray.300", "whiteAlpha.100");
+
+	const handleJustifyCenterClick = () => {
+		document.execCommand("justifyCenter", true, "");
+	};
+
+	return (
+		<Tooltip
+			label="Justify Center"
+			aria-label="Justify Center"
+			color={textColor}
+			bg={boxBg}
+		>
+			<Button color={textColor} onClick={handleJustifyCenterClick}>
+				   <MdFormatAlignJustify/>
```

Comment:
This looks bad, per your own admission. Please use such icon box instead.

```suggestion
				<IconBox
					icon={
						<Icon
							w="18px"
							h="18px"
							as={MdFormatAlignCenter}
							color={textColor}
						/>
					}
				/>
```